cdap-validation/bin/input.json

```diff
@@ -0,0 +1,10 @@
+{
+  "params":
+  {
+    "verbose": "default",
+    "uri": "http://ambari.prod.continuuity.net:8080/",
```

Comment:
are these actual hosts? don't use actual values.  Should this file be checked in at all?